shared/validation/place-order-request-validator.ts

```diff
@@ -77,6 +78,24 @@ function isNullOrWhitespace(input: string) {
     return !input || input.replace(/\s/g, '').length < 1;
 }
 
+function validateOrderTime(): string[] {
+    let errors: string[] = [];
+    let now: Date = new Date();
+    let day: number = now.getDay();
+    let time: number = now.getHours() * 100 + now.getMinutes();
+    if (time < 1200) {
+        errors.push(`Sorry, You can't place an order before 12:00 PM`);
+    }
+    if (day < 5 && time > 1300) {
+        errors.push(`Sorry, You can't place an order after 01:00 PM`);
```

Comment:
This should be 1AM, not 1PM... if it stays as it is then it would mean the shop is only open for 1 hour (hence why I think we should have tests for this validation)